HW05_Parts1_-_2.md

```diff
@@ -0,0 +1,126 @@
+  arrange(fam_id)%>%
+  rename(Family_Number = fam_id)
+
+kable(tidy_dad_mom)
+```
+
+| Family\_Number | Parent | Name | Income |
+| -------------: | :----- | :--- | :----- |
+|              1 | Dad    | Bill | 30000  |
+|              1 | Mom    | Bess | 15000  |
+|              2 | Dad    | Art  | 22000  |
+|              2 | Mom    | Amy  | 22000  |
+|              3 | Dad    | Paul | 25000  |
+|              3 | Mom    | Pat  | 50000  |
+
+\#\#Part 2, Combining Data Sources
+
+``` r
+AUC_1 <- read.csv("CTRP_files/AUC_1.csv")
+AUC_2 <- read.csv("CTRP_files/AUC_2.csv")
+cancer_cell_line_invo <- read.csv("CTRP_files/cancer_cell_line_info.csv")
+compound_info <- read.csv("CTRP_files/compound_info.csv")
+experiment_info <- read.csv("CTRP_files/experiment_info.csv")
+
+cancer_data <- AUC_1%>%
+  full_join(AUC_2, by=c("experiment_id", "area_under_curve", "master_cpd_id"))%>%
+  full_join(compound_info, by="master_cpd_id")%>%
+  full_join(experiment_info, by=c("experiment_id"="expt_id"))%>%
+  full_join(cancer_cell_line_invo, by="master_ccl_id")
+```
+
+\#\#Part 2, Question 1 The below graph summarizes the Effect of
+Vorinostat on different cancer types. You can see haematopoietic and
+lymphoid tissue has the lowest average AUC, while the lowest extreme
+comes from tissue in the upper aerodigestive tract.
+
+``` r
+low_AUC <- cancer_data%>%
+  select(experiment_id, cpd_name, cancer_type, area_under_curve)%>%
+  filter(cpd_name=="vorinostat")%>%
+  arrange(area_under_curve)%>%
+  group_by(cancer_type)
+
+ggplot(low_AUC, aes(x=reorder(cancer_type, area_under_curve,), y=area_under_curve))+
+geom_boxplot()+
+  theme(axis.text.x=element_text(angle=65, hjust=0.9))+
+  labs(x="Cancer", y="Distribution", title="Vorinostat Effect on Cancer Types")
+```
+
+![](HW05_Parts1_-_2_files/figure-gfm/question1-1.png)<!-- -->
+
+\#\#Part 2, Question 2 In this case, only one row of data was required
+to answer the question. Therefore, a simple kable table was used to show
+that the cancer cell line 22RV1 was most sensitve to compound leptomycin
+B.
+
+``` r
+prostate_sens <- cancer_data%>%
+  select(experiment_id, area_under_curve, cpd_name, ccl_name, cancer_type)%>%
+  filter(ccl_name == "22RV1")%>%
+  arrange(area_under_curve)%>%
+  top_n(n=-1, wt = area_under_curve)%>%
+  rename(Experiment=experiment_id, AUC=area_under_curve, Compound=cpd_name, Cell_Line=ccl_name, Cancer_Type=cancer_type)
+
+kable(prostate_sens)
+```
+
+| Experiment |    AUC | Compound     | Cell\_Line | Cancer\_Type |
+| ---------: | -----: | :----------- | :--------- | :----------- |
+|        139 | 2.6367 | leptomycin B | 22RV1      | prostate     |
+
+\#\#Part 2, Question 3 Based on the graph, the compound WZ8040 had the
+lowest AUC values in breast cancer cell lines.
+
+``` r
+EGFR_AUC <- cancer_data%>%
+  select(experiment_id, gene_symbol_of_protein_target, cpd_name, area_under_curve, cancer_type)%>%
+  filter(gene_symbol_of_protein_target=="EGFR", cancer_type=="breast")%>%
```

Comment:
I would argue there is a better way. As Robert pointed out in the announcement, the `gene_symbol_of_protein_target` column can contain a list of genes separated by ';'. It might be a good idea to use functions like `grepl` or `str_detect` to find them all.

Reply:
I agree, this was just a format I readily understood and could anticipate the outcome for. Given some more tinkering I am sure it could be simplified